.github/workflow/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
There is an issue with your linters not being triggered by Github Actions. Please check this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to find out the problem 🤓 Be sure to follow all the steps and check that all linters are passing on your PR before asking for another review 😉

Your PR 
![Captura](https://user-images.githubusercontent.com/15807343/153050513-e2cc344b-5506-4d14-87c9-e3302c385f79.PNG)

Github Actions 
![Captura2](https://user-images.githubusercontent.com/15807343/153050504-419255bc-73c7-4d63-a99b-d358f5fbdaa0.PNG)

Reply:
How to check Github Actions?

Reply:
Dear @amrendrakind, the previous reviewer means that your PR linter does not make any actions, so it is not checking your code - actually, it is not working and does not do any action. You should fix it. How? I will write down in this review the reason why it is not working 👍